Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/parameter_entry.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import * as Nexus from 'nexus';
+import { DisplaySize } from '../../../display_size';
+import { CurrencySelectionBox, MoneyInputBox, TextInputField, 
+  IntegerInputBox } from '../../../components';
+import { ComplianceParameter, ComplianceValue } from 'nexus';
```

Comment:
Nexus is treated as a top-level dependency, so it should come before local dependencies.